### PR TITLE
fix(SCT-471): radio button styling when selecting related person

### PR DESCRIPTION
**Pull request by chubberlisk**
**What**  

Fix radio button styling when selecting a person for a relationship.

Before | After
--|--
image | image

**Why**  

So it matches visually with other radio buttons.

**Anything else?**

- Have you added any new third-party libraries?
- Are any new environment variables or configuration values needed?
- Anything else in this PR that isn't covered by the what/why?

### CODE REVIEW

**Review by mapleleafjack — APPROVED** [review]
🔘

File: components/Search/results/RelationshipTable.tsx [hunk 1]
```diff
@@ -14,15 +14,23 @@ const ResultEntry = ({
...
+              id={mosaicId}
+              name="personId"
+              type="radio"
+              value={mosaicId}
+              onChange={(e) => {
+                callback(e.target.value);
+              }}
+              className={`govuk-radios__input`}
+            />
+            <label className="govuk-label govuk-radios__label">
```
Comment (jhackett1) [inline components/Search/results/RelationshipTable.tsx:28]: needs a `htmlFor={mosaicId}` attribute to be valid html

**Review by jhackett1 — CHANGES_REQUESTED** [review]
one minor a11y/semantic html fix

File: components/Search/results/RelationshipTable.tsx (hunk 1 above)
Comment (chubberlisk) [reply components/Search/results/RelationshipTable.tsx]: Ahhhhh, thanks for the spot!

**Review by jhackett1 — APPROVED** [review]

**Pull request merged**